src/lib/chat/matrix-client.ts

```diff
@@ -266,12 +266,22 @@ export class MatrixClient implements IChatClient {
 
     const messageResult = await this.matrix.sendMessage(channelId, content);
     const newMessage = await this.getMessageByRoomId(channelId, messageResult.event_id);
+    await this.recordMessageSent(channelId, newMessage.createdAt);
```

Comment:
Maybe we don't need to `await` here? Fire and forget might be better just to not cause issues with the app if this request ever starts failing.

Reply:
makes sense.